### ISSUE TITLE
Recursive schema is not rendering

### ISSUE THREAD

**Issue by kchaber** [asyncapi/asyncapi-react#351]
**Description**

Schema which contains some recursive items does not render and the console logs present some StackOverflow exceptions.

**Expected result**

Schema with recursive items should render correctly.

**Steps to reproduce**

To reproduce this issue you can use the following schema where `RecursiveItem` schema contains list of `subItems` of type `RecursiveItem`.

```
asyncapi: '2.0.0'
info:
  title: Recursive API
  version: '1.0.0'

channels:
  recursive-items:
    subscribe:
      message:
        $ref: '#/components/messages/RecursiveItem'

components:
  messages:
    RecursiveItem:
      payload:
        $ref: "#/components/schemas/RecursiveItem"

  schemas:
    RecursiveItem:
      type: object
      properties:
        name:
          type: string
        subItems:
          type: array
          items:
            $ref: '#/components/schemas/RecursiveItem'
```

Tested on both versions: `0.23.1` and `1.0.0-next-3`

**Comment by magicmatatjahu** [comment]
@kchaber Hello! Thanks for reporting the issue! We know about this bug, for more info check https://github.com/asyncapi/parser-js/issues/293 If we will support the recursive schemas in every place in spec (at the moment we only support in properties) then supporting in component will be very easy.

**Comment by derberg** [comment]
@magicmatatjahu what do you mean we support it in properties only? arrays too. This file works well with HTML template before migration to react

**Comment by magicmatatjahu** [comment]
Yes, but as I described in this issue https://github.com/asyncapi/parser-js/issues/293 (with example) circular refs in items aren't handled properly, so supporting in component is disabled. In most cases this will result in a wrong rendered schema.

**Comment by derberg** [comment]
not sure what you mean by `most cases` because so far it was discovered only with your only case and not even with real schema but a dummy schema for testing only 😄 

I'm not saying the bug is not there, it is there and your example shows it pretty nice, I totally agree with you.

On the other hand though, `next` react component was supposed to not introduce any regression so your approach here is not the best in my opinion. 

Technically what's the problem to support current parser functionality? I mean why not enabling it now and support `most cases` 😄 later it is just a matter of upgrading to latest parser and more cases will be supported, right?

**Comment by magicmatatjahu** [comment]
@kchaber Issue is fixed in `1.0.0-next.5`. We also published new playground which has included fixing. https://asyncapi.github.io/asyncapi-react/ Let us know if someone is broken :) Your schema is rendered as below:

![image](https://user-images.githubusercontent.com/20404945/119663465-a6bd5400-be32-11eb-83a7-08c08b0d2e1a.png)

**Comment by kchaber** [comment]
Thanks a lot! Looks ok now 🙂
